recipe-client-addon/test/browser_env_expressions.js

```diff
@@ -53,4 +53,12 @@ add_task(function* () {
   // Test stable sample returns true for matching samples
   val = yield EnvExpressions.eval('["test"]|stableSample(0)');
   is(val, false, "Stable sample returns false for 0% sample");
+
+  // Test bucket sample returns true for a known sample
```

Comment:
I like these tests!

I think we should add some tests for stableSample like these that aren't at 100% or 0%, so that we'll know in the future if we break the way input is hashed and could cause currently-running recipes to be sent to different users than they were before.